components/automate-gateway/api/iam/v2/rules.proto

```diff
@@ -66,7 +196,27 @@ service Rules {
     option (chef.automate.api.iam.policy).action = "iam:projects:update";
   };
 
+  /*
+  Starts project update
+
+  Any changes to a project's rules are staged first. They do not take effect until 
+  all projects are updated.
+
+  Updating all projects begins an operation that applies all pending rule edits 
+  and then moves ingested resources into the correct projects according to those latest changes.
+
+  With a large amount of historical compliance data, rule application can take a considerable amount of time.
+  It’s best to batch up rule changes and apply them all at once.
+
+  Authorization Action:
+  ```
+  iam:rules:apply
+  ```
+  */
   rpc ApplyRulesStart (ApplyRulesStartReq) returns (ApplyRulesStartResp) {
+    option (grpc.gateway.protoc_gen_swagger.options.openapiv2_operation) = {
```

Comment:
so these APIs render first, I think because `v2/apply-rules` comes alphabetically before `v2/projects/id/rules`. Gonna see if docs team has any ideas how to reorder them

Reply:
jk they say it's fine 💁